src/v1/interfaces.ts

```diff
@@ -286,7 +367,7 @@ export interface AddressSearcherResponse extends AddressResolverResponse {
   /**
    * The query that the search has been performed for.
    */
-  query: string;
+  query: string | AddressSearcherQuery;
```

Comment:
* As this is not exposed as a public API yet, just expecting `AddressSearcherQuery` would be fine.

Reply:
fixed